_src/dashboard.md

```diff
@@ -82,7 +82,18 @@ nav: Dashboard
     <div class="graphic-text">
        <p>Though this is a national crisis, each state is reporting data differently. We are tracking numbers from each state, though the quality and frequency of reports varies significantly.</p>
     </div>
-    <div class="graphic" id="chart-states-current-death-total"></div>
+    <div class="graphic" id="chart-states-current-death-total">
+      <h3 class="chart-hed">Total deaths by State</h3>
+      <div class="chart no-y-axis-domain"></div>
+      <div class="graphic-footer">
+        <div class="chart-expand-button"></div>
```

Comment:
If this is serving as a button, that is to say that a viewer can click it and the page will respond, we should make it an actual button so that it can be interacted with from a keyboard/assistive technology. Not worth preventing a merge, but definitely something we want to clean up soon in a subsequent PR.